gnocchi/cli.py

```diff
@@ -86,7 +86,11 @@ def change_sack_size():
     ])
     conf = service.prepare_service(conf=conf)
     s = storage.get_incoming_driver(conf.incoming)
-    report = s.measures_report(details=False)
+    try:
+        report = s.measures_report(details=False)
+    except Exception:
+        # issue is already logged by measures_report
```

Comment:
where? I skimmed through the code but I see nowhere it is handled. This should be handled here to me with a proper exception raised by the underlying driver(s).